pkg/bpf/map.go

```diff
@@ -701,3 +773,78 @@ func (m *Map) GetModel() *models.BPFMap {
 
 	return mapModel
 }
+
+// resolveErrors is schedule by scheduleErrorResolver() and runs periodically.
+// It resolves up to maxSyncErrors discrepancies between cache and BPF map in
+// the kernel.
+func (m *Map) resolveErrors() error {
+	started := time.Now()
```

Comment:
Nit: shouldn't `started` be instantiated after the lock for the map is acquired?